nf_core/pipelines/lint/nf_test_content.py

```diff
@@ -22,56 +22,51 @@ def nf_test_content(self):
     * A `versions.yml` file should be included in the snapshot of all `*.nf.test` files
 
     * The `nextflow.config` file should contain:
-        .. code-block:: yaml
+        .. code-block:: groovy
             modules_testdata_base_path = <path>
 
-        .. code-block:: yaml
+        .. code-block:: groovy
             pipelines_testdata_base_path = <path>
 
         And should set the correct resource limits, as defined in the `test` profile
 
     * The `nf-test.config` file should:
         * Make sure tests are relative to root directory
 
-            .. code-block:: yaml
+            .. code-block:: groovy
 
                 testsDir "."
 
         * Ensure a user-configurable nf-test directory
 
-            .. code-block:: yaml
+            .. code-block:: groovy
 
                 workDir System.getenv("NFT_WORKDIR") ?: ".nf-test"
 
         * Use a test specific config
 
-            .. code-block:: yaml
+            .. code-block:: groovy
 
                 configFile "tests/nextflow.config"
 
     All these checks can be skipped in the `.nf-core.yml` file using:
 
-        .. code-block:: yaml
+        .. code-block:: groovy
             lint:
                 nf_test_content: False
 
         or
 
-        .. code-block:: yaml
+        .. code-block:: groovy
             lint:
                 nf_test_content:
-                    - outdir
-                    - versions.yml
-                    - modules_testdata_base_path
-                    - pipelines_testdata_base_path
-                    - resourceLimits
-                    - testsDir
-                    - workDir
-                    - configFile
+                    - tests/<test_name>.nf.test
+                    - tests/nextflow.config
+                    - tests/nf-test.config
     """
-    passed = []
-    failed = []
-    ignored = []
+    passed: list[str] = []
```

Comment:
```suggestion
    passed: List[str] = []
```
we are still on 3.8 until https://github.com/nf-core/tools/pull/3538 is merged